podpac/datalib/drought_monitor.py

```diff
@@ -0,0 +1,36 @@
+
+from podpac.core.data.types import Zarr
+from podpac.core.coordinates import ArrayCoordinates1d
+
+class DroughtMonitorCategory(Zarr):
```

Comment:
@mpu-creare Is this enough? Do you need any sort of compositor, too?

Reply:
Nope, This looks great. Thanks!